### PR TITLE
Add cloudflare 'access' service

### PR DESCRIPTION
**Pull request by d0m0reg00dthing**
#162 
* `access`
   * `cloudflare_access_application`

### CODE REVIEW

File: providers/cloudflare/access.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+		))
+	}
+
+	return resources, nil
+}
+
+func (g *AccessGenerator) InitResources() error {
+	api, err := g.initializeAPI()
+	if err != nil {
+		log.Println(err)
```
Comment (sergeylanzman) [inline providers/cloudflare/access.go]: Not need log.println here, You return error

File: providers/cloudflare/access.go [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+		return err
+	}
+
+	zones, err := api.ListZones()
+	if err != nil {
+		log.Println(err)
```
Comment (sergeylanzman) [inline providers/cloudflare/access.go]: Same

File: providers/cloudflare/access.go [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+		return err
+	}
+
+	for _, zone := range zones {
+		tmpRes, err := g.createAccessApplications(api, zone.ID)
+		if err != nil {
+			log.Println(err)
```
Comment (sergeylanzman) [inline providers/cloudflare/access.go]: Same

File: providers/cloudflare/access.go (hunk 1 above)
Comment (d0m0reg00dthing) [reply providers/cloudflare/access.go]: Sorry, I forgot to remove it. Removed

**Pull request merged**